TEAM-MISSION.md

```diff
@@ -0,0 +1,51 @@
+# Team Mission
+
+## FLS Team - `t-fls`
+
+The initial charter of the FLS team is:
+
+> The FLS Team maintains and improves the FLS for the benefit
+> of the Project and the safety-critical community.
+> The FLS Team is responsible for:
+> 
+> * Maintaining and improving the content and tooling of the FLS.
+> * Keeping the FLS up to date with new releases of Rust.
+> * Setting and documenting guidelines for authorship and
+>   contribution to the FLS.
+> * Working with the safety-critical community to ensure the FLS
+>   continues to serve them, particularly with regard to supporting
+>   the qualification of Rust-related tools.
+> * Working with other Rust teams to ensure the accuracy of content
+>   in the FLS.
+> * Working with the maintainers of the Reference to identify areas
+>   where each document and team can provide value to the other.
+> * Cultivating and curating membership of `fls-contributors` and
+>   setting policies for that team.
+> 
+> The FLS Team leads are tasked with ensuring the health of the team.
+> The team manages its own membership and selects its own leads.
+
+## FLS Contributors Team - `t-fls-contributors`
+
+The initial charter of the FLS contributors team is:
+
+> The FLS Contributors Team supports the work of the FLS Team in
+> maintaining and improving the FLS.
+> The FLS Contributors Team is responsible for:
+> 
+> * Proposing updates and improvements to the FLS, following the
+>   guidance of the FLS Team.
+> * Working with members of the FLS Team to refine and integrate
+>   those changes.
+> * Reviewing pull requests and suggesting improvements to them.
+> * Identifying changes to the FLS relevant to the Reference and
+>   filing Reference issues or submitting Reference PRs.
+> * Triaging issues and pull requests.
+> 
+> The FLS Contributors team does not have any leads and its membership is managed by the FLS Team. The team is a subteam of the FLS Team.
+
+--
+
+*Note*: The above content pulled from the `teams` repo
+[pull request](https://github.com/rust-lang/team/pull/2028)
+which created the teams.
```

Comment:
```suggestion
```

This could go in the commit message, but it should probably be dropped from the file.  E.g., if we evolve the charter language in a later commit, this note would no longer be true in the same sense.